### ISSUE TITLE
"Error: Use -machine help to list supported machines" on start

### ISSUE THREAD

**Issue by andrew-noyo** [utmapp/UTM#3722]
**Describe the issue**
I receive the message `Error: Use -machine help to list supported machines` without a crash log.

<img width="1012" alt="Screen Shot 2022-03-04 at 7 57 03 PM" src="https://user-images.githubusercontent.com/99214790/156868017-878c3c70-eacd-42b3-8c3d-98b4d6304f3d.png">

![Uploading Screen Shot 2022-03-04 at 7.56.59 PM.png…]()

**Configuration**
* UTM Version: 3.1.3 (51)
* OS Version: 12.2.1
* Intel or Apple Silicon? Apple Silicon

**Debug log**
[debug.log](https://github.com/utmapp/UTM/files/8189975/debug.log)

**Upload VM**
`<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
	<key>ConfigurationVersion</key>
	<integer>2</integer>
	<key>Debug</key>
	<dict/>
	<key>Display</key>
	<dict>
		<key>ConsoleFont</key>
		<string>Menlo</string>
		<key>ConsoleFontSize</key>
		<integer>12</integer>
		<key>ConsoleOnly</key>
		<false/>
		<key>ConsoleTheme</key>
		<string>Default</string>
		<key>DisplayCard</key>
		<string>virtio-vga</string>
		<key>DisplayDownscaler</key>
		<string>linear</string>
		<key>DisplayUpscaler</key>
		<string>nearest</string>
	</dict>
	<key>Drives</key>
	<array>
		<dict>
			<key>DriveName</key>
			<string>cdrom0</string>
			<key>ImageType</key>
			<string>cd</string>
			<key>InterfaceType</key>
			<string>ide</string>
			<key>Removable</key>
			<true/>
		</dict>
		<dict>
			<key>DriveName</key>
			<string>drive0</string>
			<key>ImagePath</key>
			<string>data.qcow2</string>
			<key>ImageType</key>
			<string>disk</string>
			<key>InterfaceType</key>
			<string>ide</string>
		</dict>
	</array>
	<key>Info</key>
	<dict>
		<key>Icon</key>
		<string>linux</string>
	</dict>
	<key>Input</key>
	<dict/>
	<key>Networking</key>
	<dict>
		<key>NetworkCard</key>
		<string>e1000</string>
		<key>NetworkCardMAC</key>
		<string>2E:F4:A1:18:3A:B6</string>
		<key>NetworkMode</key>
		<string>shared</string>
	</dict>
	<key>Printing</key>
	<dict/>
	<key>Sharing</key>
	<dict>
		<key>ClipboardSharing</key>
		<true/>
		<key>DirectoryReadOnly</key>
		<false/>
		<key>DirectorySharing</key>
		<true/>
		<key>UsbRedirectMax</key>
		<integer>3</integer>
	</dict>
	<key>Sound</key>
	<dict>
		<key>SoundCard</key>
		<string>intel-hda</string>
		<key>SoundEnabled</key>
		<true/>
	</dict>
	<key>System</key>
	<dict>
		<key>Architecture</key>
		<string>aarch64</string>
		<key>BootDevice</key>
		<string></string>
		<key>BootUefi</key>
		<true/>
		<key>CPU</key>
		<string>default</string>
		<key>CPUCount</key>
		<integer>6</integer>
		<key>Memory</key>
		<integer>6144</integer>
		<key>RngEnabled</key>
		<true/>
		<key>SystemUUID</key>
		<string>63F81744-4DF5-490D-86B5-5F9AA5B97646</string>
		<key>Target</key>
		<string>q35</string>
		<key>UseHypervisor</key>
		<true/>
	</dict>
</dict>
</plist>
`

**Comment by osy** [comment]
How did you create this VM? Can you give a specific step by step or a screen capture? It appears you’ve created a x86_64 machine with ARM64 architecture which should never happen…

**Comment by boywijnmaalen** [comment]
I also have this error, doesn't matter which version of UTM I use (3.0.4, 3.1.0, 3.1.3-2).
all result in the same error when starting a VM. I also tried recreating the VM using different configs, still I get the "Use -machine help to list supported machines". The image I use is ARM64

**Comment by osy** [comment]
@boywijnmaalen can you answer the question I posted above?

**Comment by plaurent** [comment]
I see the same error.  Selecting an older version of UTM, e.g., [v3.1.0 (Beta)](https://github.com/utmapp/UTM/releases/tag/v3.1.0) from 20 days ago, the ISO boots fine with the default settings.

**Comment by osy** [comment]
> How did you create this VM? Can you give a specific step by step or a screen capture? It appears you’ve created a x86_64 machine with ARM64 architecture which should never happen…

PLEASE someone do this or it'll never be fixed.

**Comment by plaurent** [comment]
> > How did you create this VM? Can you give a specific step by step or a screen capture? It appears you’ve created a x86_64 machine with ARM64 architecture which should never happen…
> 
> PLEASE someone do this or it'll never be fixed.

Hi @osy ,

The error occurs while following the instructions here: https://mac.getutm.app/gallery/ubuntu-20-04

Specifically,

> ### Creating a new virtual machine
>1. Open UTM and click the “+” button to open the VM creation wizard.
>1. Select “Virtualize”.
>1. Select “Linux”.
>1. Click “Browse” and select the Ubuntu Server ISO downloaded from the link above. Press “Next” to continue.
>1. Pick the amount of RAM and CPU cores you wish to give access to the VM. Press “Next” to continue.
>1. Specify the maximum amount of drive space to allocate. Press “Next” to continue.
>1. If you have a directory you want to mount in the VM, you can select it here. Alternatively, you can skip this and select the directory later from the VM window’s toolbar. The shared directory will be available after installing SPICE tools (see below). >1. Press “Next” to continue.
>1. Press “Save” to create the VM and press the Run button to start the VM.
>1. Go through the Ubuntu Server installer. If the reboot fails, you can manually quit the VM, unmount the installer ISO, and start the VM again to boot into your new installation.

Thank you for looking into this.

**Comment by osy** [comment]
Can you provide your machine configuration and UTM version?

**Comment by boywijnmaalen** [comment]
@osy, apologies for the late reply. I used the same instructions as provided by @plaurent. I'm running a M1 Max, 64gb, not using rosetta. I think (but just assuming here) the error started occurring  when I manually installed QEMU (using Homebrew) alongside UTM. can't think of anything else that changed between a working version of UTM and an UTM having issues as described above (so at one point I did have a working UTM/Linux/virtualized setup).
